docs/account.mdx

```diff
@@ -0,0 +1,630 @@
+---
+title: Account operations
+description: Using CLI to manage your tokens
+---
+
+import DocCard from '@theme/DocCard';
+import {findSidebarItem} from '@site/src/sidebarUtils';
+
+# `account`
+
+`account` command is used for consensus or ParaTime-layer on-chain actions
+associated with accounts:
+
+- getting the account balance including delegations,
+- transferring tokens between the accounts,
+- delegating or undelegating tokens to or from validators (*staking*),
+- depositing and withdrawing tokens to or from a ParaTime,
+- changing permission policy of the account,
+- setting commission schedule of the validator account.
+
+## Show
+
+Use `account show [address]` command to print the balance of the account on the consensus layer
+and the currently selected ParaTime, if it is non-zero. Running it without
+arguments will show you the balance of your default account.
+
+```
+$ oasis account show
+Address: oasis1qrec770vrek0a9a5lcrv0zvt22504k68svq7kzve
+Nonce: 28
+
+=== CONSENSUS LAYER (testnet) ===
+  Total: 984.669098086 TEST
+  Available: 984.669098086 TEST
+
+
+  Allowances for this Account:
+    Total: 0.0000002 TEST
+    Allowances:
+      - Beneficiary: oasis1qrydpazemvuwtnp3efm7vmfvg3tde044qg6cxwzx
+        Amount:      0.0000002 TEST
+
+
+=== emerald PARATIME ===
```

Comment:
Should we have sapphire as default?

Reply:
Yeah, that would make sense both for Mainnet and Testnet now.

Reply:
https://github.com/oasisprotocol/oasis-sdk/pull/1374